src/v2/view-builder/views/email/RequiredFactorEmailView.js

```diff
@@ -9,20 +9,24 @@ const SHOW_RESEND_TIMEOUT = 60000;
 
 const ResendView = View.extend(
   {
-  // by default is hide after one minute polling
+    //only show after certain threshold of polling
     className: 'hide resend-email-view',
+    events: {
+      'click a.link-send-again' : 'handleSengAgainClick'
+    },
+
     initialize () {
       this.add(createCallout({
-        subtitle: 'Haven\'t received an email? To try again, click "Resend Email"',
+        content: `Haven\'t received an email? <a class='link-send-again'>Send again</a>`,
         type: 'warning',
       }));
-      this.add(createButton({
-        className: 'button',
-        title: 'Resend Email',
-        click () {
-          this.options.appState.trigger('invokeAction', 'factor-resend');
-        }
-      }));
+    },
+
+    handleSengAgainClick() {
+      this.options.appState.trigger('invokeAction', 'factor-resend');
+      //hide warning, but reinitiate to show warning again after some threshold of polling
+      this.$el.addClass('hide');
+      this.postRender();
```

Comment:
Yeah i was thinking about that to take it out to a new function but didn't do because we don't have anything else right now. But make sense to do it right now.